plugins/azure/virtualmachinescaleset/vmssManagedIdentityEnabled.js

```diff
@@ -0,0 +1,47 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'VM Scale Set Managed Identity Enabled',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensures that Azure Virtual Machine scale sets have managed identity enabled.',
+    more_info: 'Managed identities for Azure resources provide Azure services with an automatically managed identity in Microsoft Entra ID. You can use this identity to authenticate to any service that supports Microsoft Entra authentication, without having credentials in your code.',
```

Comment:
can we make it consistent like we have in other managed identity plugins ,
Enabling managed identities eliminate the need for developers having to manage credentials by providing an identity for the Azure resource in Azure AD and using it to obtain Azure Active Directory (Azure AD) tokens.